### ISSUE TITLE
Grails 3.3.0 constraint "editable: false" no longer works

### ISSUE THREAD

**Issue by martinfrancois** [grails/grails-core#10794]
This one is similar to #10763, but concerning the ["editable" constraint](https://docs.grails.org/latest/ref/Constraints/Usage.html).

### Steps to Reproduce

1. `grails create-app website`
`cd website`
`grails create-domain-class Sample`
`grails create-scaffold-controller website.Sample`
2. Update Sample.groovy:
```
package website
class Sample {
    String description
    static constraints = {
         description editable: false
    }
}
```
3. run: `grails run-app`
4. go to: http://localhost:8080/sample/create

### Expected Behaviour

Input field next to "Description" should be grayed out and should not be possible to be changed, displaying the current value as read only.

### Actual Behaviour

Input field next to "Description" is freely editable.

### Environment Information

- **Operating System**: Windows 10 x64
- **Grails Version:** 3.3.0
- **JDK Version:** 1.8.0_121

**Comment by graemerocher** [comment]
Duplicates #10763

**Comment by martinfrancois** [comment]
@graemerocher I don't consider it to be a duplicate really, since this one concerns the "editable" constraint, while #10763 is concerning the "textarea" constraint. Since other (even scaffolding) constraints, such as "display" still work, I think it likely won't be fixed by fixing the other issue.

**Comment by graemerocher** [comment]
The fix is the same as #10763 

For reference this line fixes this issue: https://github.com/grails/grails-core/commit/ed432f2c2ee95f07633714289e07c6eba34feeee#diff-7dc2e9016dd72e25bffd03d02fad0384R192

**Comment by martinfrancois** [comment]
Ah, I didn't know that, thanks for clearing it up!
